### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
2 new templates under .github directory. No impact to run-time.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (lubaihua33) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: I don't understand this. Could you add some details?

Comment (juhlee-microsoft) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: This is Github template for bug or issue entry. Please check out wala agent repo, https://github.com/azure/walinuxagent

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
```
Comment (LiliDeng) [inline .github/ISSUE_TEMPLATE/bug_report.md]: why we need this?

**Review by LiliDeng — CHANGES_REQUESTED** [review]

**Review by abhimarathe — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (juhlee-microsoft) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed

**Comment by juhlee-microsoft** [comment]
No need to fit into LISAv2 work.

**Pull request closed without merging**